runtime/jcl/common/sun_misc_Unsafe.cpp

```diff
@@ -833,7 +833,11 @@ Java_jdk_internal_misc_Unsafe_objectFieldOffset1(JNIEnv *env, jobject receiver,
 		if (NULL == romField) {
 			vmFuncs->setCurrentExceptionUTF(currentThread, J9VMCONSTANTPOOL_JAVALANGINTERNALERROR, NULL);
 		} else if (J9_ARE_ANY_BITS_SET(romField->modifiers, J9AccStatic)) {
-			vmFuncs->setCurrentExceptionUTF(currentThread, J9VMCONSTANTPOOL_JAVALANGILLEGALARGUMENTEXCEPTION, NULL);
+			offset = fieldID->offset | J9_SUN_STATIC_FIELD_OFFSET_TAG;
```

Comment:
We should check when this new behaviour was introduced and add a macro to keep the old behaviour for older releases

Reply:
tested locally on linux build, this behavior has been like this since the API it is introduced in Java10

Reply:
okay we shuold add JAVA11+ flags to this then